### PR TITLE
Fixes issue #718

### PR DESCRIPTION
**Pull request by wcsanders1**
Here is a checklist you should tick through before submitting a pull request: 
 - [x] Implementation is clean
 - [x] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [x] No ReSharper warnings
 - [x] There is proper unit test coverage
 - [ ] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [x] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [ ] Xml documentation is added/updated for the addition/change
 - [x] Your PR is (re)based on top of the latest commits from the `dev` branch (more info below)
 - [x] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [ ] Readme is updated if you change an existing feature or add a new one
 - [x] Run either `build.cmd` or `build.ps1` and ensure there are no test failures

### CODE REVIEW

**Comment by wcsanders1** [comment]
I think the changes in this PR will resolve issue 718. Let me know if there's anything else I need to do with regard to this PR.

File: src/Humanizer.Tests/Humanizer.Tests.csproj [hunk 1]
```diff
@@ -24,9 +24,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <ContentWithTargetPath Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (onovotny) [inline src/Humanizer.Tests/Humanizer.Tests.csproj]: This should be PreserveNewest or it breaks incremental builds. Why was this changed?

File: src/Humanizer.Tests/Humanizer.Tests.csproj [hunk 2]
```diff
@@ -24,9 +24,10 @@
...
+      <TargetPath>PublicApiApprovalTest.approve_public_api.approved.txt</TargetPath>
```
Comment (onovotny) [inline src/Humanizer.Tests/Humanizer.Tests.csproj]: Why did this need to change?

File: src/Humanizer.Tests/Humanizer.Tests.csproj (hunk 1 above)
Comment (wcsanders1) [reply src/Humanizer.Tests/Humanizer.Tests.csproj]: I made this change because the ApiApproved test was failing and I thought that this change would fix it. I was wrong. Reverted in e968e34.

File: src/Humanizer.Tests/Humanizer.Tests.csproj (hunk 2 above)
Comment (wcsanders1) [reply src/Humanizer.Tests/Humanizer.Tests.csproj]: The ApiApprover test was failing and it appeared to me that the reason for this was that the approved.txt file was not being copied to the directory where the test runner expected it to be. I made this change, which appeared to solve the problem. However, after reverting the change, the test still passes, so I guess the change was unnecessary. Reverted in 947466a.

**Comment by onovotny** [comment]
Please update the `readme.md` with the new functionality and this will be good to merge. Thanks!

**Comment by wcsanders1** [comment]
Readme updated in 54a060e.
